mgz/fast.py

```diff
@@ -15,6 +15,7 @@ class Operation(Enum):
 
 class Action(Enum):
     """Action types."""
+    UNKNOWN = -1
```

Comment:
My preference is always to raise an exception for actual parse problems. If an unknown action is encountered, it should be explicitly added to the enum (as something like `DE_UNKNOWN_X`).